### ISSUE TITLE
Can't modify $usage_str to add a command line argument

### ISSUE THREAD

**Issue by kablamo** [celogeek/MooX-Options#32]
So options are nicely handled.  But not bare arguments.  For example, I can't have a usage string like:

    USAGE: cmd subcmd FILE DIR [-h] [long options...]

Where FILE can be any filename and DIR can be any DIR.  Would you accept a pr for this?  I am thinking it would be configured like this:
```
use MooX::Options 
    description => '...', 
    args => 'FILE DIR';
```

**Comment by celogeek** [comment]
Yes, I can accept.

They is an option to allow additional option like that, the flavour params.

I think if we have "args" on the use, we can also activate the flavour, so it would work as expected.

**Comment by celogeek** [comment]
Thanks.
released into v4.018
